src/game/client/draw/drawable.cpp

```diff
@@ -26,6 +28,18 @@ const Matrix3D *Drawable::Get_Transform_Matrix() const
     return Thing::Get_Transform_Matrix();
 }
 
+bool Drawable::Is_Visible() const
+{
+    const DrawModule **draw_modules = Get_Draw_Modules();
+
+    for (const DrawModule *draw_module = draw_modules[0]; draw_modules != nullptr; ++draw_module) {
+        if (draw_module->Is_Visible()) {
```

Comment:
I did not see the vtable mappings of `DrawModule` class, but guessed `Is_Visible` looks like the right choice.

Reply:
yeah correct